msticpy/data/queries/m365d/kql_mdatp_process.yaml

```diff
@@ -22,15 +22,19 @@ defaults:
       description: Additional query clauses
       type: str
       default: ""
+    time_column:
+      description: The name of the column detailing the time the event was generated.
+      type: str
+      default: "Timestamp"
 sources:
   list_host_processes:
     description: Lists all process creations for a host
     metadata:
     args:
       query: '
         {table}
-        | where Timestamp >= datetime({start})
-        | where Timestamp <= datetime({end})
+        | where {time_column} >= datetime({start})
```

Comment:
time_column seems a bit of weird name - maybe time_stamp?

Reply:
So I picked this as we use time column (or time_col) everywhere else we parameterize the column name such as in the plots, and so thought it would keep things consistent.